picturae_csv_create.py

```diff
@@ -427,6 +350,112 @@ def check_if_images_present(self):
 
         self.record_full['image_valid'] = self.record_full['image_path'].apply(lambda path: os.path.exists(path))
 
+
+    def check_taxa_against_database(self):
```

Comment:
made new function, using the .apply function to check taxonomic full names against database *before* sending to TNRS, to avoid having to check everything with tnrs. TNRS can be slow if fed a ton of names.